zyte_api/_retry.py

```diff
@@ -177,3 +177,62 @@ def build(self) -> AsyncRetrying:
 
 
 zyte_api_retrying: AsyncRetrying = RetryFactory().build()
+
+
+def is_maybe_temporary_error(exc: BaseException) -> bool:
```

Comment:
_maybe_temporary_error? This looks more similar to other functions like this in this module.